__tests__/ClusterManagement.js

```diff
@@ -208,12 +180,64 @@ details view`, async () => {
 
   // Expect we have been redirected to the cluster details view
   expect(getAllByText(V4_CLUSTER.id)[0]).toBeInTheDocument();
+});
+
+it(`redirects the user to clusters to list and shows flash message when cluster doesn't exist`, async () => {
+  const fakeCluster = 'f4ke1';
+
+  getMockCall('/v4/organizations/', orgsResponse);
+  getMockCall('/v4/clusters/');
+
+  const clusterDetailPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.Detail,
+    {
+      orgId: ORGANIZATION,
+      clusterId: fakeCluster,
+    }
+  );
+  const { getByTestId } = renderRouteWithStore(clusterDetailPath);
+
+  // Expect we have been redirected to the clusters list.
+  await wait(() => expect(getByTestId('clusters-list')).toBeInTheDocument());
+
+  const flashMessage = document.querySelector('#noty_layout__topRight');
+  expect(flashMessage).toContainHTML(
+    `Cluster <code>f4ke1</code> doesn't exist.`
+  );
+});
+
+it.only('Cluster list shows all clusters, each one with its details, for the selected organization', async () => {
```

Comment:
I think you forgot this here 😬 

```suggestion
it('Cluster list shows all clusters, each one with its details, for the selected organization', async () => {
```

Reply:
🤦‍♂